tests/Feature/IndexTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\User;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Facades\Artisan;
+use Laravel\Passport\Passport;
+use Tests\TestCase;
+
+class IndexTest extends TestCase
+{
+    use DatabaseMigrations;
+    protected function setUp():void
+    {
+        parent::setUp();
+        Artisan::call('migrate');
+        Artisan::call('db:seed');
+        Artisan::call('passport:install');
+    }
+
+    public function test_index()
+    {
+        User::factory()->count(5)->create();
+
+        $this->get('/api/users')->assertStatus(200);
```

Comment:
```suggestion
        $this->get('/api/users')->assertStatus(200)->assertJsonStructure(['users']); // since we have a rule to turn back structured data, you can also check its structure, just a tip :)
```